libraries/SFU/extra/loader/loader.ino

```diff
@@ -0,0 +1,207 @@
+/*
+  Portenta C33 - 2nd stage bootloader
+
+  The sketch checks for available update files on QSPI system partion
+  or SD card and try to update the Sketch
+
+  Build (release) with arduino-cli:
+    # arduino-cli compile --export-binaries --clean --verbose -b arduino-git:renesas:portenta_c33 --build-property "build.extra_flags=-DNO_USB -DPORTENTA_C33_SFU_BUILD"
+  Build (debug) with arduino-cli:
+    # arduino-cli compile --export-binaries --clean --verbose -b arduino-git:renesas:portenta_c33 --build-property "build.extra_flags=-DNO_USB -DPORTENTA_C33_SFU_DEBUG_OTA -DPORTENTA_C33_SFU_BUILD"
```

Comment:
Once released, this sketch can be also built using the platform installed from the board manager, but I think it will be mainly used during the development phase. After making some changes in the core or in the SFU library  the developer will rebuild the loader using its manually installed platform and then exports the binary into the [c33.h](https://github.com/arduino/ArduinoCore-renesas/pull/116/files#diff-6ff879f917792244f69acf89686d53534565e488edc470a879c0a898b5e3b545) file to relase the new loader.